File: test/ocamlTests/BoolTest/fromIntTest.ml
test "fromInt(0)" (fun () -> expect (Bool.fromInt 0) |> toEqual (let open Eq in option bool) Some(false)) ; 
test "fromInt(1)" (fun () -> expect (Bool.fromInt 1) |> toEqual (let open Eq in option bool) Some(true)) ; 
test "fromInt(Int.minimumValue)" (fun () -> expect (Bool.fromInt Int.minimumValue) |> toEqual (let open Eq in option bool) None) ; 
test "fromInt(-2)" (fun () -> expect (Bool.fromInt -2) |> toEqual (let open Eq in option bool) None) ; 
test "fromInt(-1)" (fun () -> expect (Bool.fromInt -1) |> toEqual (let open Eq in option bool) None) ; 
test "fromInt(2)" (fun () -> expect (Bool.fromInt 2) |> toEqual (let open Eq in option bool) None) ; 
test "fromInt(Int.maximumValue)" (fun () -> expect (Bool.fromInt Int.maximumValue) |> toEqual (let open Eq in option bool) None) ; 

File: test/fsharpTests/BoolTest/fromIntTest.fs
testCase "fromInt(0)" 
<| fun _ -> 
    let expected = Some(false)
    Expect.equal expected (Bool.fromInt 0) "error"
testCase "fromInt(1)" 
<| fun _ -> 
    let expected = Some(true)
    Expect.equal expected (Bool.fromInt 1) "error"
testCase "fromInt(Int.minimumValue)" 
<| fun _ -> 
    let expected = None
    Expect.equal expected (Bool.fromInt Int.minimumValue) "error"
testCase "fromInt(-2)" 
<| fun _ -> 
    let expected = None
    Expect.equal expected (Bool.fromInt -2) "error"
testCase "fromInt(-1)" 
<| fun _ -> 
    let expected = None
    Expect.equal expected (Bool.fromInt -1) "error"
testCase "fromInt(2)" 
<| fun _ -> 
    let expected = None
    Expect.equal expected (Bool.fromInt 2) "error"
testCase "fromInt(Int.maximumValue)" 
<| fun _ -> 
    let expected = None
    Expect.equal expected (Bool.fromInt Int.maximumValue) "error"
